### ISSUE TITLE
Data augmentation with noise resulted in  unreasonable performance improvement on ResNet50

### ISSUE THREAD

**Issue by szlge** [keras-team/keras-applications#134]
I trained a ResNet50 using the GTSRB data set (http://benchmark.ini.rub.de/?section=gtsrb&subsection=news), then I "augmented" the previously employed GTSRB data set by randomly generated images of the appropriate dimension and repeated the training procedure. Finally I observed that the "noise-augmented" training procedure resulted in an unreasonable +20% accuracy improvement on the same validation data set. Clearly It is not reasonable, we should expect worse accuracy in the "augmented" case. 
This situation holds regardless of the number of epochs. The same situation has also been observed on other data sets, such as CIFAR-10.

![case01](https://user-images.githubusercontent.com/19731435/63273816-f276bd00-c29e-11e9-90ba-f77ae4ba77a6.jpg)

![case02](https://user-images.githubusercontent.com/19731435/63273831-f99dcb00-c29e-11e9-9aa0-a0b9e3512283.jpg)

**Comment by ketyi** [comment]
Gergo, could you try an evaluation of the model just by the noise_array training data exclusively? How would that go?

**Comment by szlge** [comment]
In such a cases I ended up with an accuracy close to 0, as it is expected.

**Comment by ketyi** [comment]
At least, that's good.
